source/manual/govuk-notify.html.md

```diff
@@ -33,6 +33,15 @@ for each environment):
   example might be to receive an email when a scheduled document on GOV.UK has
   been automatically published.
 
+Currently (as of 30/06/2020) GOV.UK Notify has a maximum rate limit of 350
```

Comment:
This only applies to "GOV.UK Emails". Can we move it as a note under that ☝️ ?

Reply:
Sure